bokeh/core/compat/__init__.py

```diff
@@ -24,3 +24,9 @@
 .. _mplexporter: https://github.com/mpld3/mplexporter/tree/master/mplexporter
 
 '''
+from ...util.deprecation import deprecated
+
+deprecated(
+    "MPL compatibility can no longer be successfully maintained, and is now deprecated. " +
+    "All MPL compat functions will be removed completely on the release of Bokeh 1.0. " +
```

Comment:
nothing where? I can guarantee nothing inside the project does. My definite preference is to give folks as long as possible and also for the breakpoint for such a big deletion to be "logical"